### DISCUSSION TITLE
Is psPAS supported with Privilege Cloud?

### DISCUSSION THREAD

**Discussion by marc-fom** [pspete/psPAS#479] [General]
Hi,

Sorry if this information is provided somewhere and I missed it but trying to connect psPAS module to our Privilege Cloud baseURL I'm getting errors whatever I try with New-PASSession command.
I'm using CyberArk type connection and tried different baseURL:
https://subdomain.privilegecloud.cyberark.cloud
https://subdomain.cyberark.cloud 

So does psPAS support only self-hosted PAM or Privilege Cloud is also supported?

Please advise.

Best
Marc

**Comment by pspete** [comment]
As of right now, you can use these instructions: https://pspas.pspete.dev/docs/authentication/#shared-services-authentication

Soon we may be able to offer additional options in the module.

**Comment by marc-fom** [comment]
Hi Pete,

Thanks, this moved me one step further.
Using the creds that work properly for my custom built PowerShell provisioning script does not with your module, this is the error I’m getting:
Invoke-PASRestMethod : invalid client creds or client not allowed

What are the roles or account attributes that you expect?
Does it have to be a service user? Guess so from your wording “Provide tenant ID and non-interactive API User credentials for authentication”
The user I’m passing in creds is indeed a service user.

Best regards
Marc

**Reply by pspete** [reply]
Refer to the Vendor documentation on the API service user: https://docs.cyberark.com/PrivCloud-SS/Latest/en/Content/ISPSS/ISPSS-API-Authentication.htm
